src/platform/plugins/shared/dashboard/public/dashboard_api/types.ts

```diff
@@ -168,4 +173,5 @@ export interface DashboardInternalApi {
   isSectionCollapsed: (sectionId?: string) => boolean;
   dashboardContainerRef$: BehaviorSubject<HTMLElement | null>;
   setDashboardContainerRef: (ref: HTMLElement | null) => void;
+  serializeControls: () => SerializeControlsResult;
```

Comment:
Does `serializeControls` need to be added to dashboardApi? Its not used any where.

Reply:
It is used [here](https://github.com/elastic/kibana/pull/236740/files#diff-ca4daab3cdca04e5952c031511a66188556d09118fa0ff32a17705c52896f368R88).